src/silly_sprites/script/builtins.cpp

```diff
@@ -0,0 +1,47 @@
+#include "builtins.hpp"
+#include <iostream>
+#include <spdlog/spdlog.h>
+
+void print(std::string const& text) {
+    std::cout << text;
+}
+
+void println(std::string const& text) {
+    std::cout << text << '\n';
+}
+
+void eprint(std::string const& text) {
+    std::cerr << text;
+}
+
+void eprintln(std::string const& text) {
+    std::cerr << text << '\n';
+}
+
+namespace logging {
+
+    void trace(std::string const& text) {
```

Comment:
for what do we need those and would it be good to make it a varidaic template.
so we could pass some arguments.
but i dont know the context. maybe thats stupid.

Reply:
Those are not meant to be called from within C++, but from AngelScript. They are exposed to AngelScript in the `Log::` namespace, so that you can write something like `Log::debug("The number is " + number);` in AngelScript. Because AngelScript supports string concatenation, I don't (yet?) think that using a variadic template would be useful here (and I don't even know how to use a templated function across both languages).

Reply:
never mind then